src/server/server.ts

```diff
@@ -7,6 +7,10 @@ import http from 'http';
 import dotenv from "dotenv";
 import path from 'path';
 import { apiRouter } from './routers/api.routes.js'
+import { RoleModel } from "./schemas/role.schema.js";
+import { UserModel } from "./schemas/user.schema.js";
+import { createUser } from "./routers/user-route.js";
+import { nextTick } from "process";
```

Comment:
I would also suggest to remove this line to minimize code "noise".

Reply:
good catch! it's deleted now